src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+# Revolution Rework
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| illdoodle | :x: No | TBD |
+
+## Overview
+***THIS CONCEPT IS TRANSLATED WITH “DEEPL TRANSLATE”, there is maybe translation errors***
+
+Also it worth noting, that this is just _concept_. Unfortunately, i don't know C#
+
+As many people know at the moment revolution gamemode is actually a legalized LRP slashing, which for many players for obvious reasons causes resentment, 
+unwillingness to play the mode and even thoughts of its complete removal from the game.
+upd: it’s already deleted :pepefrog:
+
+On some SS13 servers for exactly the same reasons the mode is simply taken out of rotation and we should not exclude that in time the same fate awaits it in SS14, when people will get tired of the mess as it is now. 
+For these reasons it is necessary for the mode to conduct a complete revamp of mechanics (code), rules, lore.
+
+Also worth noting are the current flashes, which are a definite meta, because the player even by sound becomes aware of the game mode, which can affect his future decisions, which the admins obviously can't track. 
+Also flashes are an easy excuse to choke the headrevs and ruin the round.
+The forced conversion mech itself is stupid. People won't even have a choice of which side to step on, which will narrow the RP process in the mode and just turn it into a prototype of cultists, 
+getting rid of any meaning and mood of the rebellion itself.
+
+Anyways the goal of this concept is to not treat revolution as a LRP mode, thereby increasing its RP level and lowering player carelessness. 
+It's also worth considering revolution as a major mode rather than a minor mode.
+
+### Concept Abstracts
+- unique scoring system
+- voluntary entry into the revolution
+- rev icon replacement
+- additional avenues for recruitment
+- rev uplink 
+- various rebel paraphernalia
+- cult-like alert
+
+## Background
+**You can read a more comfortable and beautiful version here --->** <https://docs.google.com/document/d/1KaitkDoAZFSBrNIc78K522gtLkDHlZaMzH9vNMe9Oz4/edit?usp=sharing>
+
+It's contains additional comments, illustatrations, images and other MAYBE useful information.
+
+**RUS VERSION --->** <https://docs.google.com/document/d/11Lpip1kL9xyVAVoQn-PVuqBHv-stovz8ZaweRDGwxtE/edit?usp=sharing>
+
+## 1. GENERAL MECHANICS
+### 1.1 Rev Uplink
+- Headrevs have another distinctive feature, the rev **uplink implant**. It contains a variety of often decrepit, rusty and makeshift equipment.
+- For **one recruited** it gives **1 Rebel Telecrystal** (RTC) to headrev’s uplink.
+> A full list of gear is being worked on.
+Approximately:
+Soviet-made pistols, Mosinky, PPSh, Kalashnikovs, RGDs, various body armor, clothing, riot posters, homemade war makings, ammunition and other paraphernalia
```

Comment:
This is great! I love the idea of Nanotrassen trying to stop a socialist "commie" uprising. Stylistically it's a great nod to the red revolution (Plus it ties in nicely to the syndicate with their red themeing)